diff --git a/node_modules/node-html-parser/dist/nodes/html.js b/node_modules/node-html-parser/dist/nodes/html.js
index 8969f3a..04b2be6 100644
--- a/node_modules/node-html-parser/dist/nodes/html.js
+++ b/node_modules/node-html-parser/dist/nodes/html.js
@@ -1112,19 +1112,8 @@ function parse(data, options) {
         var oneBefore = (0, back_1.default)(stack);
         if (last.parentNode && last.parentNode.parentNode) {
             if (last.parentNode === oneBefore && last.tagName === oneBefore.tagName) {
-                // Pair error case <h3> <h3> handle : Fixes to <h3> </h3>
-                oneBefore.removeChild(last);
-                last.childNodes.forEach(function (child) {
-                    oneBefore.parentNode.appendChild(child);
-                });
-                stack.pop();
             }
             else {
-                // Single error  <div> <h3> </div> handle: Just removes <h3>
-                oneBefore.removeChild(last);
-                last.childNodes.forEach(function (child) {
-                    oneBefore.appendChild(child);
-                });
             }
         }
         else {
